src/ddimm_parser.cpp

```diff
@@ -221,6 +221,13 @@ size_t
 void DdimmVpdParser::readKeywords(
     types::BinaryVector::const_iterator i_iterator)
 {
+    if (m_vpdVector.empty() ||
```

Comment:
This is a private API of this class, why this vital check is required at this point?

Reply:
Removed this check, as other check is sufficient even for zero size vector.